themes/default/content/docs/guides/automation-api/concepts-terminology.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Core Concepts and Terminology
+meta_desc: This page contains an overview of core concepts when using Automation API.
+linktitle: Automation API Concepts
+weight: 1
+
+menu:
+  userguides:
+    parent: automation-api
+---
+
+Automation API lets you define a Pulumi program as a function within your codebase rather than in a separate project and use methods to get and set configuration parameters programmatically.
+
+The package can be used for a number of use cases:
+
+- Driving pulumi deployments within CI/CD workflows
+- Integration testing
+- Multi-stage deployments such as blue-green deployment patterns
+- Deployments involving application code like database migrations
+- Building higher level tools, custom CLIs over pulumi, etc
+- Using pulumi behind a REST or GRPC API
+- Debugging Pulumi programs (by using a single main entrypoint with "inline" programs)
+
+## Workspace
+
+To enable a broad range of runtime customization the API defines a Workspace interface. A Workspace is the execution context containing a single Pulumi project, a program, and multiple stacks. Workspaces are used to manage the execution environment, providing various utilities such as plugin installation, environment configuration (`$PULUMI_HOME`), and creation, deletion, and listing of Stacks.
+
+Custom Workspace implementations can be used to store Project and Stack settings as well as Config in a different format, such as an in-memory data structure, a shared persistent SQL database, or cloud object storage. Regardless of the backing Workspace implementation, the Pulumi SaaS Console will still be able to display configuration applied to updates as it does with the local version of the Workspace today.
+
+### LocalWorkspace
+
+A default implementation of workspace is provided as LocalWorkspace. This implementation relies on `Pulumi.yaml` and `Pulumi.[stack].yaml` as the intermediate format for Project and Stack settings. Modifying `ProjectSettings` will alter the Workspace `Pulumi.yaml` file, and setting config on a Stack will modify the `Pulumi.[stack].yaml` file. This is identical to the behavior of Pulumi CLI driven workspaces.
```

Comment:
Not sure if I changed the meaning of the first sentence, so let me know if it's not quite right.

```suggestion
The default implementation of a workspace is known as a LocalWorkspace. This implementation relies on `Pulumi.yaml` and `Pulumi.[stack].yaml` as the intermediate format for Project and Stack settings. Modifying `ProjectSettings` will alter the workspace `Pulumi.yaml` file, and setting config on a stack will modify the `Pulumi.[stack].yaml` file. This is identical to the behavior of Pulumi CLI-driven workspaces.
```